lib/components/STable.vue

```diff
@@ -175,6 +183,10 @@ function isSummary(index: number) {
   return index === records?.value?.length
 }
 
+function lastRow(index: number) {
+  return index === recordsWithSummary.value.length - 1
```

Comment:
This too can be avoided if we can get the last row using CSS. From what I can tell, the order of rows isn't same in resultant HTML at least initially. But if the scroll is at the bottom, then the last row in HTML was actually the last. So might be possible using CSS only 👀